### ISSUE TITLE
Create a Beeminder integration

### ISSUE THREAD

**Issue by Stvad** [n8n-io/n8n#62]
Beeminder is a service that helps you to achieve your goals by providing accountability: https://beeminder.com

API: http://api.beeminder.com/#beeminder-api-reference

**Comment by janober** [comment]
@stvad Do you have planned to work on that yourself? If so all is fine and we can leave the ticket here. If it is however only a suggestion that somebody else should create it please post it here: https://community.n8n.io/c/feature-requests/nodes 
That is the place to add integration requests. Other users can then upvote them if they are also interested. That gives me than the chance to know what is in demand and should be integrated next. Thanks!

**Comment by Stvad** [comment]
I'm considering looking into this one myself, but likely not in the next few weeks at least. I'll duplicate the request on the link, but FYI you can sort issues by the number of emoji reactions they have, which should give you an idea about their popularity (i.e. https://github.com/n8n-io/n8n/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc current sorting demonstrates that people are passionate about the language 😛)

**Comment by Stvad** [comment]
Created https://community.n8n.io/t/beeminder-integration/143

**Comment by janober** [comment]
@Stvad Great thanks a lot. Close the issue then here for now. 
Ah yes, you are right could also be done here. The problem would then just be that there would be suddenly two locations to do the same thing and that would make things very confusing. So just trying to keep things organized and so hopefully easy and clear.

**Comment by Stvad** [comment]
sure, but just from my perspective github issues seems like a far more natural place to request the integrations 😉  so I suspect that you'd have a whole bunch of issues like this 😛

**Comment by janober** [comment]
@Stvad Not sure if it's more natural. The name "issue" suggests more that something is wrong or broken. But agree that it would definitely not be the wrong place. It is simply something I did decide because not all users of n8n will be technical so I can not assume that they have Github account. So it seemed like a good idea to add it to the forum. Also, are none technical people even more likely to request an integration (as they can not create one themself) than developers are. So allowing them to ask questions and request integration in the same tool they are already familiar with because they ask other questions seemed reasonable. I did not want to force them to additionally create a Github account and "learn" how Github works.
In case you wonder why I created some integration requests myself in the issues. I only did that to be part of Hacktoberfest. After it is over I will remove them again.
I want to create an issue template (hopefully later in the day) in which I mention that people should add integration requests to the forum instead. So I hope to reduce it like that to a minimum.
